src/core/Logo/index.js

```diff
@@ -2,15 +2,17 @@
 
 import React from 'react'
 import PropTypes from 'prop-types'
-import { bemClassNames } from '../../utils'
+import { componentStyles } from '../../utils'
 
 import './styles.css'
 import iconBlue from './svg/icon-blue.svg'
 import iconWhite from './svg/icon-white.svg'
 import logoBlue from './svg/logo-blue.svg'
 import logoWhite from './svg/logo-white.svg'
 
-const bem = bemClassNames('logo')
+import './styles.css'
+import styles from './styles.json'
```

Comment:
This JSON file contains the mapping from `.logo` to `._logo_1234xyz`. It's possible to enable `camelCase` so it's possible to just use `styles.logoIcon`.

```
// styles.json 
{
  "logo-icon": "._logo-icon_116zl_1",
  "logoIcon": "._logo-icon_116zl_1"
}
```